From c8d9df500b1ca83ad6da9b6055549adaba6ee0d3 Mon Sep 17 00:00:00 2001
From: Olivier Singla <olivier.singla@gmail.com>
Date: Thu, 4 Aug 2016 22:05:08 +0200
Subject: [PATCH] launcher.c: include <sys/ttydefaults.h>

This include is needed to get the definition of TTYDEF_*.

Signed-off-by: Olivier Singla <olivier.singla@gmail.com>
---
 shellinabox/launcher.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/shellinabox/launcher.c b/shellinabox/launcher.c
index 2bac171..c8ba6cb 100644
--- a/shellinabox/launcher.c
+++ b/shellinabox/launcher.c
@@ -66,6 +66,7 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <sys/utsname.h>
+#include <sys/ttydefaults.h>
 #include <termios.h>
 #include <unistd.h>
 
-- 
2.7.4

